src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -69,6 +78,17 @@ public StorageFile(String filePath) throws InvalidStorageFilePathException {
             throw new InvalidStorageFilePathException("Storage file should end with '.txt'");
         }
     }
+    
+
+    /**
+     * Check whether the storage file exists at the path
+     * @throws FileNotExistException
+     */
+    public void checkFileExist() throws FileNotExistException{
+    	if(!new File(path.toString()).exists()) {
+    		throw new FileNotExistException("Storage file doesn't exist");
+    	}
```

Comment:
So what happens in this method if file does exist? That is, what is the normal control flow?